upgrade-notes/backend_20251111_135924.md

```diff
@@ -8,4 +8,6 @@
 - `Shopsys\FrameworkBundle\Model\Order\Mail` class was changed:
     - `getDisplayPrice()` method was moved to the new `MailDisplayPriceResolver` class
     - `DISPLAY_PRICE_WITH_VAT`, `DISPLAY_PRICE_WITHOUT_VAT`, `DISPLAY_PRICE_BOTH`, and `DISPLAY_PRICE_SELLING` constants were moved to the new `MailDisplayPriceResolver` class
+- `Shopsys\FrontendApiBundle\Model\Order\OrderItemApiFacade::createOrderItemQueryBuilder()` was renamed to `createOrderItemExcludingOrdersWithWithdrawalQueryBuilder()`
+    - the method now excludes orders with withdrawal requests from the result
```

Comment:
nitpick - this method always excluded orders with withdrawal requests as no withdrawal requests existed before

Reply:
true, removed in 828bcde40d